jobs/rocoto/fcst.sh

```diff
@@ -12,9 +12,10 @@ source "${HOMEgfs}/ush/preamble.sh"
 source "${HOMEgfs}/ush/detect_machine.sh"
 set +x
 source "${HOMEgfs}/ush/module-setup.sh"
-module use "${HOMEgfs}/sorc/ufs_model.fd/tests"
-module load modules.ufs_model.lua
-module load prod_util
```

Comment:
You're still missing these module loads.

Reply:
Fixed.